src/subscription_manager/scripts/rhsmcertd_worker.py

```diff
@@ -259,6 +259,19 @@ def _auto_register_standard(uep: "UEPConnection", token: Dict[str, str]) -> None
     service = RegisterService(cp=uep)
     service.register(org=None, jwt_token=token["token"])
 
+    # When the system is configured to manage repositories, then install entitlement
+    # certificates and generate the redhat.repo file
+    manage_repos = config.get_config_parser().get("rhsm", "manage_repos")
```

Comment:
I see from other places in the code they do `from subscription_manager.repofile import manage_repos_enabled` and use that method to determine its value (which also uses a get_int, which seems more type safe?). Is there a reason not to use that one?